drivers/entropy/Kconfig

```diff
@@ -11,6 +11,7 @@ config ENTROPY_CC310
 	depends on ENTROPY_GENERATOR
 	depends on !BT_LL_SW_LEGACY
 	depends on !BT_LLL_VENDOR_NORDIC
+	depends on !MPU_STACK_GUARD
```

Comment:
This was a temporary fix at the time.
The correct fix is this: https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1356/files

Reply:
apparently this file lacks the disabling of cc310:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/f3859825e2bba3f696bf9fd8e1b225c714851bdf/applications/nrf_desktop/configuration/nrf52840_pca20041/app_ZDebugMCUBoot.conf#L248

Reply:
Added backport label to this PR:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1369